Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
This file has been added by accident, can we remove it

Reply:
ups, removed